wiki/Article_Styling_Criteria/en.md

```diff
@@ -274,20 +424,28 @@ Do not: https://osu.ppy.sh/home
 Do not: <https://osu.ppy.sh/home>
 Do not: [https://osu.ppy.sh/home](https://osu.ppy.sh/home)
 
-Do: [osu! on the web](https://osu.ppy.sh/home)
+Do:     [osu! on the web](https://osu.ppy.sh/home)
 ```
 
 #### Internal
 
-Internal links are the links linking within the osu! website.
+Internal links are the links linking within the website.
 
 ##### Wiki links
 
-All links that point to an osu! wiki article must start with `/wiki/` followed by the path to get to the article you are targeting.
+All links that point to an osu! wiki article must start with `/wiki/` followed by the path to get to the article you are targeting. For example:
 
-You could use a redirect to shorten a link a bit further.
+```
+[Welcome](/wiki/Welcome)
+```
+
+You could use a redirect to shorten a link a bit further. For example:
+
+```
+[BN](/wiki/BN)
+```
 
-##### Other osu! web pages
+##### Other osu! webpages
```

Comment:
L448: This doesn't seem right. Explain why would you remove the spacing between web and pages.

Reply:
Should it be `osu!web pages`?